Source/Csla/Polyfills/RangeIndexPolyfill.cs

```diff
@@ -0,0 +1,204 @@
+﻿//-----------------------------------------------------------------------
+// <copyright file="RangeIndexPolyfill.cs" company="Marimer LLC">
+//     Copyright (c) Marimer LLC. All rights reserved.
+//     Website: https://cslanet.com
+// </copyright>
+// <summary>Polyfill implementation of Range and Index for target frameworks where they are unavailable</summary>
+//-----------------------------------------------------------------------
+#if NET462
+namespace System;
```

Comment:
Same as in the allow null PR. Polyfill nuget has to be updated and this file is obsolete.